303-verification-workflows.md

```diff
@@ -0,0 +1,32 @@
+---
+title: CNAB Security: Verification workflows
+weight: 303
+---
+
+# Cloud Native Application Bundles Security (CNAB-Sec): Verification workflows 1.0 WD
+
+- [Verification workflows](#verification-workflows)
+
+This document is a _prescriptive_ part of [CNAB Security](300-CNAB-security.md).
+
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119). The use of these keywords in this document are statements about how a CNAB implementation may fulfill the CNAB Security specification _only_.
+
+## Verification workflows
+
+This section describes the verification workflow that runtimes compliant with the CNAB Security Specifications MUST perform in order to verify the provenance and attestation of bundles. Regardless of [the bundle format][bundle-formats], the verification workflows is the same, with the signature of a thin bundle being the content digest of the `bundle.json` file, while the signature of a thick bundle being the content digest of the bundle archive.
+
+End-users SHOULD see no difference in their experience, unless an attack is caught. In that case, the installation of the bundle in question MUST be denied, and users SHOULD see an error message indicating why the verification workflow failed.
+
+In order to download and verify a bundle, a compliant runtime MUST take the following steps:
+
+1. Use the [TUF workflow][tuf-workflow] to download trusted hashes about the desired bundle (e.g., `example.com/example-org/example-bundle:latest`) from the [metadata repository][metadata-repository].
```

Comment:
I would use _download trusted [metadata]_ instead of _hashes_, but it's not blocking.